Credits.txt

```diff
@@ -0,0 +1,18 @@
+This file should be updated with every addition to the SoundFX library.
+
+Credits for samples:
+
+Karst van Galen Last (AuraSpecs):
+news, pop, rain, flush, thunder_1, thunder_2, thunder_3, thunder_4, hydrolics_tow_truck, hydrolics_wh_lost_gravity, hydrolics_el_abyssus, hydrolics_wh_space_shot, scream_el_zadra, 
+train_whistle_1_cp_railroad, train_whistle_2_cp_railroad, train_whistle_3_cp_railroad, train_whistle_depart_cp_railroad, bm_rc_lift_cp_rougarou, bm_rc_roar_cp_rougarou,
+wood_rc_lift_cp_bluestreak, scream_el_zadra, hyper_rc_lift_cp_magnum_xl_200,
+
+Ted John (IntelOrca):
+crowd, 
+
+(ThatGuyYouKnow):
+hydrolics_ninja,
+
+All samples listed above are recorded for OpenRCT2 and are licensed under CC BY-SA (https://creativecommons.org/licenses/by-sa/3.0/)
```

Comment:
The license currently in the repo is the CC-BY-SA 4.0 license, not the 3.0 license. 
```suggestion
All samples listed above are recorded for OpenRCT2 and are licensed under CC BY-SA 4.0 (https://creativecommons.org/licenses/by-sa/4.0/)
```

Reply:
As I found out, this is actually licensed under CC BY 4.0. A PR to change this is now open (#9).